ports/devel/xsd/Makefile.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/devel/xsd/Makefile b/devel/xsd/Makefile
```

Comment:
The location for diffs is in `diffs` within the port, not in the port directory itself, see other ports as an example.